### PR TITLE
#485 - Document CSS conventions and update CSS around

### LINKED ISSUE #485: Document CSS conventions on Group Income
### Problem
At the moment is not clear (we don't have documentation) how we deal with CSS on Vue component at Group Income.
- Use Bulma classes whenever possible
- Global Group Income CSS classes are prefixed with `.g-`
- Scoped CSS classes on a specific Vue component are prefixd with `c.-`

### Solution
- Write documentation explaining it in detail with working examples how we do it.
- Refactor the existing Vue CSS that doesn't follow the conventions.

### PR DESCRIPTION
**Pull request by sandrina-p**
- Improve Style Guide - CSS part
- Update some of the CSS around to follow the `.gi-` vs `.c-` class conventions

- Scout rule: Remove unused `containers/YourGroupsList.vue`

Closes #485

### CODE REVIEW

**Comment by taoeffect** [comment]
@sandrina-p as there are some merge conflicts now in this PR and I'm not sure which CSS prefix is correct (e.g. `.gi-` vs `.c-`) I want to play it safe and allow you to resolve them instead of me guessing via the GitHub UI.

File: docs/Style-Guide.md [hunk 1]
```diff
@@ -76,29 +114,33 @@ $message-radius: 1px;
...
+  &.gi-is-unfilled {
+    border-style: dashed
   }
 }
 ```
 
-When's used it's painless to differentiate our code from Bulma's!
-
 ```html
+```html
```
Comment (taoeffect) [inline docs/Style-Guide.md]: Line 142 is a duplicate and should be deleted

File: docs/Style-Guide.md [hunk 2]
```diff
@@ -76,29 +114,33 @@ $message-radius: 1px;
...
 .message-body {
-  &.gi-is-box {
-    border-width: 1px;
+  // The default `border-width` value is `0 0 0 3px`.
+  // We want it to be `1px` everywhere, so we add a new modifier:
+  border-width: 1px;
+
+  // The default border-style is solid,
+  // We want to dashed it just on a particular view, so we add a new modifier:
+  &.gi-is-unfilled {
```
Comment (taoeffect) [inline docs/Style-Guide.md:135]: Is this supposed to be `.gi-is-box` to match the HTML class example below?

File: frontend/simple/views/containers/YourGroupsList.vue [hunk 3]
```diff
@@ -1,64 +0,0 @@
...
-    groups: Array,
-    currentGroupId: String
-  },
-  methods: {
-    changeGroup (hash) {
-      this.$store.commit('setCurrentGroupId', hash)
-    }
-  }
-}
-</script>
```
Comment (taoeffect) [inline frontend/simple/views/containers/YourGroupsList.vue:64]: This file is supposed to be deleted?

File: docs/Style-Guide.md (hunk 2 above)
Comment (sandrina-p) [reply docs/Style-Guide.md:135]: yes well spotted, now both are `.gi-is-unfilled`

File: docs/Style-Guide.md (hunk 1 above)
Comment (sandrina-p) [reply docs/Style-Guide.md]: removed

File: frontend/simple/views/containers/YourGroupsList.vue (hunk 3 above)
Comment (sandrina-p) [reply frontend/simple/views/containers/YourGroupsList.vue:64]: Yes, I've noticed it's not being used, it got lost when we did #436

**Comment by sandrina-p** [comment]
@taoeffect I've updated the code and did a last review, if everything's cool for you feel free to merge it :)

**Pull request merged**